src/main/java/org/apereo/openlrs/storage/noop/NoOpTierTwoStorage.java

```diff
@@ -45,6 +45,12 @@ public OpenLRSEntity findById(String id) {
 	}
 
 	@Override
+	public OpenLRSEntity findBySourceId(String sourceId) {
+		log.warn("Using NoOp Tier Two Storage");
```

Comment:
If this is always returning null value, it might be good to specify inside the log message or the function description.